src/components/Account.tsx

```diff
@@ -19,40 +19,47 @@ import People from '@material-ui/icons/People';
 import React from 'react';
 import styled from 'styled-components';
 
+import useBalance from '../hooks/useBalance';
+
 interface Props {
   value: string;
   text: string;
   className?: string;
   showDerivedBalance?: boolean;
 }
 
-// TO-DO: To replace fake balances by real balances
-const Account = ({ className, text, value, showDerivedBalance = false }: Props) => (
-  <Container className={className}>
-    <div className="address">
-      <div className="topAddress">
-        <People />
-        <div className="text">
-          <p>{text}</p>
+const Account = ({ className, text, value, showDerivedBalance = false }: Props) => {
+  const results = useBalance(value, true);
+  const source = results[0];
+  const target = results[1];
+
+  return (
+    <Container className={className}>
+      <div className="address">
+        <div className="topAddress">
+          <People />
+          <div className="text">
+            <p>{text}</p>
+          </div>
+        </div>
+        <div className="bottomAddress">
+          <p>{value}</p>
         </div>
       </div>
-      <div className="bottomAddress">
-        <p>{value}</p>
-      </div>
-    </div>
 
-    <div className="balances">
-      <div className="balance">
-        <p>0.1 RLTO</p>
-      </div>
-      {showDerivedBalance && (
+      <div className="balances">
         <div className="balance">
-          <p>0.1 MLAU</p>
+          <p>{source ? source[0] : '-'}</p>
```

Comment:
i think that if the balance would be an object instead of an array this would be easier to read. like:
`<p>{source ? source.balance : '-'}</p>`